### PR TITLE
Destination Mssql: install normalizatoin

### LINKED ISSUE #25496: Implement in-destination normalization for mssql
same process as https://github.com/airbytehq/airbyte/issues/25495

### PR DESCRIPTION
**Pull request by edgao**
closes https://github.com/airbytehq/airbyte/issues/25496

strict encrypt test failure:
```
MssqlStrictEncryptDestinationAcceptanceTest > testIncrementalDedupeSync() FAILED
    java.lang.NullPointerException: Cannot invoke "String.toLowerCase()" because "this.normalizationIntegrationType" is null
```

:shrug:

### CODE REVIEW

File: docs/integrations/destinations/mssql.md [hunk 1]
```diff
@@ -142,6 +143,7 @@ Using this feature requires additional configuration, when creating the source.
```
Comment (evantahler) [inline docs/integrations/destinations/mssql.md:143]: Why have 2 changelogs?  They should be the same... Ugh.

File: docs/integrations/destinations/mssql.md [hunk 2]
```diff
@@ -116,6 +116,7 @@ Using this feature requires additional configuration, when creating the source.
 
 | Version | Date       | Pull Request                                               | Subject                                                                                             |
 |:--------|:-----------|:-----------------------------------------------------------|:----------------------------------------------------------------------------------------------------|
+| 0.1.24  | 2023-06-05 | [\#27034](https://github.com/airbytehq/airbyte/pull/27034) | Internal scaffolding update                                                                         |
```
Comment (evantahler) [inline docs/integrations/destinations/mssql.md:119]: Maybe we should have the same release note message: `Internal code change for future development (install normalization packages inside connector)`

**Review by evantahler — APPROVED** [review]
Approved, assuming the test is failing for the same reason on master

File: docs/integrations/destinations/mssql.md (hunk 1 above)
Comment (edgao) [reply docs/integrations/destinations/mssql.md:143]: yeah these changelogs are _wildly_ different. Must have been a different time.... 😓 
image

**Comment by edgao** [comment]
good call, triggered action here https://github.com/airbytehq/airbyte/actions/runs/5182958524

**Comment by edgao** [comment]
action still in-flight, but already seeing the same error. Checking the `/test` checkbox even though tests are technically faling.
image

**Pull request merged**